cg/services/order_validation_service/rules/case_sample/rules.py

```diff
@@ -339,3 +341,15 @@ def validate_well_position_format(order: OrderWithCases, **kwargs) -> list[WellF
                 error = WellFormatError(case_index=case_index, sample_index=sample_index)
                 errors.append(error)
     return errors
+
+
+def validate_tube_container_name_unique(
+    order: OrderWithCases, **kwargs
+) -> list[ContainerNameRepeatedError]:
+    errors: list[ContainerNameRepeatedError] = []
+    for case_index, case in order.enumerated_new_cases:
+        repeated_tube_name_indices: list[str] = get_repeated_tube_names(case=case)
```

Comment:
I think we need to check this for all samples within the order so I am unsure if we can pass the case here?